marshmallow_dataclass/__init__.py

```diff
@@ -270,6 +270,9 @@ def field_for_schema(
     >>> field_for_schema(Enum("X", "a b c")).__class__
     <class 'marshmallow_enum.EnumField'>
 
+    >>> field_for_schema(Union[int,str]).__class__
```

Comment:
You need to import `Union` if you want to use it.

Reply:
@alextremblay : You have to import [Union](https://docs.python.org/3/library/typing.html#typing.Union) from typing.

Reply:
fixed :)